packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -175,4 +177,18 @@ export default abstract class EasCommand extends Command {
     await this.analytics.flushAsync();
     return super.finally(err);
   }
+
+  protected baseErrorMessage = 'Command failed.';
```

Comment:
Maybe there's a way to access the actual command name from some oclif internals?

Reply:
Haven't seen that so far, but I first heard about `oclif` yesterday 🤷  so maybe there is

Reply:
I think this should be `this.id`.
<img width="524" alt="Screenshot 2023-03-10 at 14 16 02" src="https://user-images.githubusercontent.com/5256730/224325539-a36931f9-3e86-4726-88a0-48d6ac5e2fd0.png">

Reply:
Looks like you're right, updating